.editorconfig

```diff
@@ -30,7 +30,7 @@ indent_size = 2
 indent_size = unset
 
 # Don't try to format binary files
-[*.tfvars.enc]
+[*.{tfvars.enc,ipynb}]
```

Comment:
Do you really need to unset everything, or just indentation maybe?

This particular case is for binary files.